### PR TITLE
Feature/update bridge deploy tool

### PR DESCRIPTION
**Pull request by weilbith**
Closes trustlines-network/project#595

### CODE REVIEW

File: tools/bridge-deploy/bridge_deploy/cli.py [hunk 1]
```diff
@@ -27,6 +30,16 @@ def validate_address(ctx, param, value):
         ) from e
 
 
+def validate_percentage_value(ctx, param, value):
+    if value >= 0 and value <= 100:
+        return value
+
+    else:
+        raise click.BadParameter(
+            f"The percentage value is not in the valid range of 0 till 100"
```
Comment (saschagoebel) [inline tools/bridge-deploy/bridge_deploy/cli.py]:
```suggestion
            f"The percentage value is not in the valid range of 0 to 100"
```

File: tools/bridge-deploy/bridge_deploy/cli.py [hunk 2]
```diff
@@ -35,6 +48,25 @@ def validate_address(ctx, param, value):
...
+    type=str,
+    required=True,
+    callback=validate_address,
+)
+
+validators_required_percent_option = click.option(
+    "--validators-required-percent",
+    help=(
+        "The relative amount of validators required to confirm a bridge transfer"
+        "(positive integer in range 0 till 100)"
```
Comment (saschagoebel) [inline tools/bridge-deploy/bridge_deploy/cli.py]:
```suggestion
        "(positive integer in range 0 to 100)"
```

File: tools/bridge-deploy/bridge_deploy/core.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from typing import Dict
+from web3 import Web3
+from web3.contract import Contract
+from deploy_tools.deploy import deploy_compiled_contract
+
+from bridge_deploy.utils import load_build_contract
```
Comment (saschagoebel) [inline tools/bridge-deploy/bridge_deploy/core.py]: Suggestion: As this is the only load method we're going to use in this tool, maybe rename to `load_contract`, or go the full `load_contracts_json` route as seen in the auction deploy tool.

File: tools/bridge-deploy/tests/test_cli.py [hunk 4]
```diff
@@ -10,18 +10,87 @@ def runner():
...
-def test_deploy_foreign(runner):
+def test_deploy_foreign(runner, abitrary_address):
+    result = runner.invoke(
+        main, args=f"deploy-foreign --jsonrpc test --token-address {abitrary_address}"
+    )
+
+    assert result.exit_code == 0
+
+
+def test_print_foreign(runner):
```
Comment (saschagoebel) [inline tools/bridge-deploy/tests/test_cli.py]: Strong Suggestion: This is still not implemented, right? Maybe just remove the print function & test alltogether?

File: tools/bridge-deploy/bridge_deploy/core.py (hunk 3 above)
Comment (weilbith) [reply tools/bridge-deploy/bridge_deploy/core.py]: Good point. Also obsolete part. I would move the whole function into the core module. There is nothing to share between multiple modules anymore. :+1:

File: tools/bridge-deploy/tests/test_cli.py (hunk 4 above)
Comment (weilbith) [reply tools/bridge-deploy/tests/test_cli.py]: Sound good. Just don't wanted to remove something not related to this PR.

File: tools/bridge-deploy/bridge_deploy/cli.py (hunk 1 above)
Comment (weilbith) [reply tools/bridge-deploy/bridge_deploy/cli.py]: I was searching for a word that describes that 100 is also included. So to seem to be better. :+1:

Comment (weilbith) [reply tools/bridge-deploy/bridge_deploy/cli.py]: :heavy_check_mark:

File: tools/bridge-deploy/bridge_deploy/cli.py (hunk 2 above)
Comment (weilbith) [reply tools/bridge-deploy/bridge_deploy/cli.py]: :heavy_check_mark:

File: tools/bridge-deploy/bridge_deploy/core.py (hunk 3 above)
Comment (weilbith) [reply tools/bridge-deploy/bridge_deploy/core.py]: :heavy_check_mark:

File: tools/bridge-deploy/tests/test_cli.py (hunk 4 above)
Comment (weilbith) [reply tools/bridge-deploy/tests/test_cli.py]: :heavy_check_mark:

**Pull request merged**